<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameGlobal</class>
 <widget class="QFrame" name="FrameGlobal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Engine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbEngine">
       <item>
        <property name="text">
         <string>BeeDNN</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Problem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbProblem">
       <item>
        <property name="text">
         <string>Regression</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Classification</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbData">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
